### PR TITLE
K8s client upgrade 6.0.1 to 7.0.0

### PR DESCRIPTION
**Pull request by tushar-tushar**
Version command: HyScale instead of Hyscale.

### CODE REVIEW

File: commons/src/main/java/io/hyscale/commons/constants/ToolConstants.java [hunk 1]
```diff
@@ -43,7 +43,7 @@
     
     public static final String LINUX_FILE_SEPARATOR = "/";
 
-    public static final String HYSCALE = "Hyscale";
+    public static final String HYSCALE = "HyScale";
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/constants/ToolConstants.java]: Existing paths get effected.

File: deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/impl/V1PodHandler.java [hunk 2]
```diff
@@ -251,15 +253,26 @@ public InputStream tailLogs(ApiClient apiClient, String serviceName, String name
...
-		}
+		} finally {
+		    apiClient.setHttpClient(existingHttpClient);
+        }
+	}
+	
+	private OkHttpClient getUpdatedHttpClient(OkHttpClient existingHttpClient) {
+	    Builder newClientBuilder = existingHttpClient.newBuilder()
+	            .readTimeout(120, TimeUnit.MINUTES);
+
```
Comment (SameeraGrandhi) [inline deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/impl/V1PodHandler.java]: Why is this applicable only for pods and not for any other resource ?

Comment (tushar-tushar) [reply deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/impl/V1PodHandler.java]: This is case of tailing pod logs, hence a larger timeout, in other cases we are continuing with default timeout

Comment (SameeraGrandhi) [reply deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/impl/V1PodHandler.java]: Why is this in the upgraded version of the client and not in previous versions?

File: commons/src/main/java/io/hyscale/commons/constants/ToolConstants.java (hunk 1 above)
Comment (tushar-tushar) [reply commons/src/main/java/io/hyscale/commons/constants/ToolConstants.java]: This is only used in version command, for directories we have defined it in SetupConfig.
Will rename it to HYSCALE_CAMELCASE?

**Pull request merged**